ios/FluentUI.Demo/FluentUI.Demo/Demos/NavigationControllerDemoController.swift

```diff
@@ -154,6 +164,29 @@ class NavigationControllerDemoController: DemoController {
 
         return controller
     }
+    
+    private func changeStyle(in navigationItem: UINavigationItem) {
+        DispatchQueue.main.asyncAfter(deadline: .now() + 1.0) {
+            var newStyle = navigationItem.navigationBarStyle
+            switch (newStyle) {
+            case .custom:
+                newStyle = .default
+                break;
```

Comment:
you shouldn't need break. do you have SwiftLint installed? it should have given you Xcode warnings

Reply:
no ; in swift code too

Reply:
/Users/runner/work/fluentui-apple/fluentui-apple/ios/FluentUI.Demo/FluentUI.Demo/Demos/NavigationControllerDemoController.swift:171:13: error: Control Statement Violation: `if`, `for`, `guard`, `switch`, `while`, and `catch` statements shouldn't unnecessarily wrap their conditionals or arguments in parentheses. (control_statement)
35
/Users/runner/work/fluentui-apple/fluentui-apple/ios/FluentUI.Demo/FluentUI.Demo/Demos/NavigationControllerDemoController.swift:174:22: error: Trailing Semicolon Violation: Lines should not have trailing semicolons. (trailing_semicolon)
36
/Users/runner/work/fluentui-apple/fluentui-apple/ios/FluentUI.Demo/FluentUI.Demo/Demos/NavigationControllerDemoController.swift:177:22: error: Trailing Semicolon Violation: Lines should not have trailing semicolons. (trailing_semicolon)
37
/Users/runner/work/fluentui-apple/fluentui-apple/ios/FluentUI.Demo/FluentUI.Demo/Demos/NavigationControllerDemoController.swift:38:1: error: Trailing Whitespace Violation: Lines should not have trailing whitespace. (trailing_whitespace)
38
/Users/runner/work/fluentui-apple/fluentui-apple/ios/FluentUI.Demo/FluentUI.Demo/Demos/NavigationControllerDemoController.swift:96:1: error: Trailing Whitespace Violation: Lines should not have trailing whitespace. (trailing_whitespace)
39
/Users/runner/work/fluentui-apple/fluentui-apple/ios/FluentUI.Demo/FluentUI.Demo/Demos/NavigationControllerDemoController.swift:138:1: error: Trailing Whitespace Violation: Lines should not have trailing whitespace. (trailing_whitespace)
40
/Users/runner/work/fluentui-apple/fluentui-apple/ios/FluentUI.Demo/FluentUI.Demo/Demos/NavigationControllerDemoController.swift:167:1: error: Trailing Whitespace Violation: Lines should not have trailing whitespace. (trailing_whitespace)
41
/Users/runner/work/fluentui-apple/fluentui-apple/ios/FluentUI.Demo/FluentUI.Demo/Demos/NavigationControllerDemoController.swift:185:1: error: Trailing Whitespace Violation: Lines should not have trailing whitespace. (trailing_whitespace)
42
/Users/runner/work/fluentui-apple/fluentui-apple/ios/FluentUI.Demo/FluentUI.Demo/Demos/NavigationControllerDemoController.swift:174:17: error: Unneeded Break in Switch Violation: Avoid using unneeded break statements. (unneeded_break_in_switch)
43
/Users/runner/work/fluentui-apple/fluentui-apple/ios/FluentUI.Demo/FluentUI.Demo/Demos/NavigationControllerDemoController.swift:177:17: error: Unneeded Break in Switch Violation: Avoid using unneeded break statements. (unneeded_break_in_switch)
44
/Users/runner/work/fluentui-apple/fluentui-apple/ios/FluentUI.Demo/FluentUI.Demo/Demos/NavigationControllerDemoController.swift:180:17: error: Unneeded Break in Switch Violation: Avoid using unneeded break statements. (unneeded_break_in_switch)
45
/Users/runner/work/fluentui-apple/fluentui-apple/ios/FluentUI.Demo/FluentUI.Demo/Demos/NavigationControllerDemoController.swift:183:17: error: Unneeded Break in Switch Violation: Avoid using unneeded break statements. (unneeded_break_in_switch)

Reply:
would be good to run this as part of the build, right? :)

Reply:
it is! and part of PR ci loop! if you have SwiftLint installed on your machine it is in contributed guidelinehttps://github.com/microsoft/fluentui-apple#swift-lint

Reply:
oh snaps it does :)